cross/libaom/Makefile

```diff
@@ -27,6 +27,9 @@ include ../../mk/spksrc.cross-cmake.mk
 # libaom specific cmake options
 CMAKE_ARGS += -DCONFIG_AV1_ENCODER=1
 
+# Disable tests
+CMAKE_ARGS += -DENABLE_TESTS=0
```

Comment:
Was it the tests that did not like being run in parallel? Otherwise, I wouldn't mind keeping them

Reply:
No, not related to parallel build (although at first I thought so).  With the usage of ninja all build failures went away.
I removed it as it reduces the compile time significantly (about 10%), went from 480+ down to 440ish ninja build actions.  Feel free to re-enable as you see fit.  Personally I have no need for it.